Development.xcconfig

```diff
@@ -0,0 +1 @@
+OTHER_SWIFT_FLAGS=$(inherited) -Xfrontend -warn-long-expression-type-checking=400
```

Comment:
does `400` give us mostly no warnings?

Reply:
Still about 10 or so :(

Reply:
Ah I was gonna say we should raise it till the warnings go away and then we can lower if we wanna do another pass later

Reply:
Ah, it's not too noisy at 400, though, and it only shows up for `make xcodeproj`. Maybe it'll motivate us to optimize a few along the way :)